File: pentagonalnumber/pentagonalnumber.cpp
#include <iostream>
#include <string>
using namespace std;

int PentagonalNumber(int num) {
   
    
    int sumation=1;
    for(int i=1;i<num;i++){
         int prod=i*5;
        sumation+=prod;

    }
     
  return sumation; 
            
}

int main() { 
  
  // keep this function call here
	do
	{
	int figure;
	cout<<"Input the figure :"<<endl;
	cin>>figure;

  	cout <<"output : "<< PentagonalNumber(figure)<<endl;
	
	} while (true);
  return 0;
}  

File: pentagonalnumber/pentagonalnumber.py

def get_value(value):
    
    sumation=1
    for i in range(value):
        prod=5*i 
        sumation+=prod


    return sumation
def main():
    while 1:
        value=int(input("Enter polygon value : "))
        print("value of the polygon equals %s " % get_value(value))


if __name__ == '__main__':
    main()